plugins/template-validator.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const validator = require('screwdriver-template-validator');
+
+exports.register = (server, options, next) => {
+    server.route({
+        method: 'POST',
+        path: '/validator/template',
+        config: {
+            handler: (request, reply) => {
+                // rip eslint
+                const result = validator(request.payload.yaml)
+                .then((stuff) => {
+                    reply(stuff);
+                }, (validationErr) => {
+                    reply({
+                        error: validationErr
```

Comment:
Point of feedback: What does the error case here look like for the caller?

The validator for a `screwdriver.yaml` returns a default configuration file when an error is encountered. We don't have a good default configuration for an invalid `sd-template.yaml` file.